src/core/ComboBox/ComboBox.tsx

```diff
@@ -128,6 +137,14 @@ export const ComboBox = <T,>(props: ComboBoxProps<T>) => {
   const menuRef = React.useRef<HTMLUListElement>(null);
   const toggleButtonRef = React.useRef<HTMLSpanElement>(null);
   const [isOpen, setIsOpen] = React.useState(false);
+  // used to dismiss scrolling into view on first menu open
+  // if virtualization is enabled
+  const isFirstRender = React.useRef<boolean>(false);
```

Comment:
So the menu is opened and this becomes true? But it's still true on subsequent renders so the name is a bit confusing.

Reply:
its false, when after 1st menu open focused element is reached in `menuItems` :D

Reply:
Is it still a problem?